src/data-workspace/data-details-sidebar/basic-information-follow-up-button.js

```diff
@@ -56,8 +59,21 @@ const FollowUpButton = ({ item }) => {
         )
     }
 
+    if (!canUserEditFields) {
+        return (
+            <Tooltip
+                content={i18n.t(
+                    'You do not have the authority to mark this value for follow -up'
+                )}
+            >
+                {markForFollowUpButton}
+            </Tooltip>
```

Comment:
I believe this won't work. The issue is that once a form element (incl. the `button` tag) is `disabled` it stops emitting events. Since the `button` covers the entire tooltip wrapper, the `mousenter` and `mouseleave` events will not fire and the tooltip won't show.

To deal with this type of situation the `Tooltip` from @dhis2/ui supports render-props and you can do something like the last example here: https://codesandbox.io/s/disabled-component-with-tooltip-ubtyk?file=/src/App.js

Reply:
See note in the PR's description